My Simple App/Services/AddressStore/AddressObject.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  AddressObject.swift
+//  My Simple App
+//
+//  Created by Erick Sanchez on 8/12/20.
+//  Copyright © 2020 Erick Sanchez. All rights reserved.
+//
+
+import Foundation
+
+struct AddressObject: Codable {
```

Comment:
😯 neat! I haven't come across this convention at Google, yet. But, that might be because I've only dug so far into other team's codebases and haven't passed by their "adapter" types

I think I'll keep this as-is